### PR TITLE
Enables manila tests

### PR DESCRIPTION
**Pull request by sayalilunkad**
Enables manila tests in tempest if manila service is running.

### CODE REVIEW

File: chef/cookbooks/tempest/recipes/config.rb [hunk 1]
```diff
@@ -248,6 +248,8 @@
 use_ceilometer = $?.success?
 %x{#{keystonev2} endpoint-get --service database &> /dev/null}
 use_trove = $?.success?
+%x{#{keystonev2} endpoint-get --service share &> /dev/null}
```
Comment (rsalevsky) [inline chef/cookbooks/tempest/recipes/config.rb]:
```ruby
`#{keystonev2} endpoint-get --service share &> /dev/null}`
```

Please do it for the others as well in a separate commit.

Comment (dirkmueller) [inline chef/cookbooks/tempest/recipes/config.rb]: should be unrelated cleanup that can be in a different PR..

**Comment by dirkmueller** [comment]
+1, assuming that it passes testing (which I guess it won't since we need to do more to enable manila)

File: chef/cookbooks/tempest/recipes/config.rb (hunk 1 above)
Comment (vuntz) [inline chef/cookbooks/tempest/recipes/config.rb]: There was a PR about this recently, so maybe this should be changed for the new line here too?

**Comment by vuntz** [comment]
So gating passed, but no manila tests were run in tempest.

Are some of them smoketests? Or do we need a full tempest run?

File: chef/cookbooks/tempest/recipes/config.rb (hunk 1 above)
Comment (sayalilunkad) [inline chef/cookbooks/tempest/recipes/config.rb]: Done

**Comment by sayalilunkad** [comment]
@vuntz tempest plugin for manila has not been enabled yet in the package so the tests will run.

**Comment by vuntz** [comment]
@sayalilunkad ah, okay. I'm a bit reluctant to merge this without seeing a run with the manila tests. Or did you have a run on your cloud? If yes, were the results okay?

(code-wise, +1, btw)

**Comment by sayalilunkad** [comment]
@vuntz Nope @toabctl said they won't pass :-) That's the reason we haven't enabled the plugin in the package yet.

**Comment by vuntz** [comment]
Ah, okay. If the strategy is to merge this, and then enable the tempest in the packages later on once they pass, then I'm fine merging.

**Comment by mmnelemane** [comment]
+1

**Comment by aplanas** [comment]
+1
